src/components/Restaurant/AccountButton.tsx

```diff
@@ -12,7 +13,21 @@ import Reanimated, {
 const AnimatedTouchableOpacity = Reanimated.createAnimatedComponent(TouchableOpacity);
 const AnimatedView = Reanimated.createAnimatedComponent(View);
 
-const AccountButton = ({ account, isSelected, onPress, colors }) => {
+interface LESvariebles {

```

Comment:
```suggestion
interface LESvariables {
```
Plutôt mettre vari**a**bles au lieu de vari**e**bles

Reply:
oups j'ai codé ça trop vite 😅